src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -0,0 +1,79 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines the class Wedge3D.
+
+#pragma once
+
+#include <array>
+#include <memory>
+#include <pup.h>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Direction.hpp"
+
+namespace CoordinateMaps {
+
+/*!
+ * \ingroup CoordinateMaps
+ *
+ *  \brief Three dimensional map from the cube to a wedge.
+ *
+ *  \detail The wedge is shaped such that six of them put together
+ *  make up an object where the outer surface is spherical.
+ *  The shape of the inner object can be chosen to be cubical,
+ *  if the inner_sphericity is set to 0, or to a sphere, if
+ *  the inner_sphericity is set to 1. The inner_sphericity can
+ *  be set to any number between 0 and 1 for a bulged cube.
+ */
+class Wedge3D {
+ public:
+  static constexpr size_t dim = 3;
+
+  explicit Wedge3D(double inner_radius, double outer_radius,
```

Comment:
change `inner_radius` to `cube_radius` and `outer_radius` to `sphere_radius`

Reply:
Maybe I misunderstand, but ... why? Don't we want to use these maps to make spherical shells where both the inner and outer surfaces are spherical?